doc/tutorials/validation.md

```diff
@@ -150,6 +151,22 @@ This [mounts](/doc/tutorials/mount.md) the backend `tutorial.dump` to the mount
 The validation plugin now uses the metadata of the keys below **/tests/spec**
 to validate values before storing them in `tutorial.dump`.
 
+If we try setting the key again, we will get an error:
+
+```sh
+kdb set user:/tests/spec/test "not a number"
+# STDERR: .*Validation Syntactic.*Not a number.*
+# ERROR:  C03100
+# RET     5
```

Comment:
Here is a syntax error. It needs to be RET: and only one space afterwards.

Reply:
Thank you for the remark!